app.js

```diff
@@ -1,14 +1,38 @@
 const path = require("path");
 const bodyParser = require("body-parser");
 const express = require("express");
+const mongoose = require("mongoose");
 
 const app = express();
 const expressWs = require("express-ws")(app);
 
+const javaScriptUtils = require("./app/utils/javascript-utils/javascript-utils");
 const runtimeVariables = require("./configs/runtime-variables");
-const Settings = require("./app/settings/settings");
+const Setting = require("./app/db/setting");
```

Comment:
```suggestion
const Settings = require("./app/db/settings");
```

Plural is better imo, since it stores all the settings.

Reply:
I have heard the convention of using singular nouns in the case of SQL tables. Obviously, this is neither an actual table nor SQL - however, the usage of schemas is roughly the same as defining tables(or not?). 

Shouldn't we apply this here? I'm just rather curious about the topic instead of laying down false statements. Btw agreed, `Setting` doesn't fit in the context at all.

Reply:
You're probably right, let's leave it as is. I think it depends on the framework. I'd say tables are plural but the related ORM classes are singular.